saga/seata-saga-engine/src/main/java/io/seata/saga/engine/config/AbstractStateMachineConfig.java

```diff
@@ -279,101 +280,75 @@ protected void loadStateRouterInterceptors(Map<String, StateRouter> stateRouterM
                     if (interceptor.match(interceptableStateRouter.getClass())) {
                         interceptableStateRouter.addInterceptor(interceptor);
                     }
-
-                    if (interceptor instanceof ApplicationContextAware) {
-                        ((ApplicationContextAware) interceptor).setApplicationContext(getApplicationContext());
-                    }
                 }
             }
         }
     }
 
+    /**
+     * Init StateLogStore by subClass
+     *
+     * @return StateLogStore
+     */
+    public abstract StateLangStore initStateLogStoreStore() throws Exception;
+
+    /**
+     * Init StateLogStore by subClass
+     *
+     * @return StateLogStore
+     */
+    public abstract StateLogStore initStateLogStore() throws Exception;
+
```

Comment:
I dont get the point of having these two abstract methods. I see the only implementation to these two methods merely return null. 

P.S., there is a typo. Should change `initStateLogStoreStore` to `initStateLangStore` in method signature and code comments